mlir/lib/Conversion/VectorToLLVM/ConvertVectorToLLVM.cpp

```diff
@@ -121,6 +121,25 @@ LogicalResult getVectorToLLVMAlignment(const LLVMTypeConverter &typeConverter,
   return success();
 }
 
+// Helper to resolve the alignment for vector load/store, gather and scatter
+// ops. First, this method will try to obtain the preferred alignment from the
+// load or store operation itself. If the store or load operation does not
+// contain any preferred alignment, then it will get alignment attribute through
+// the type or the backend.
+template <class LoadOrStoreOp>
+LogicalResult getVectorToLLVMAlignment(LoadOrStoreOp loadOrStoreOp,
+                                       const LLVMTypeConverter &typeConverter,
+                                       VectorType vectorType,
+                                       MemRefType memrefType, unsigned &align,
+                                       bool useVectorAlignment) {
+  if (auto alignment = loadOrStoreOp.getAlignment()) {
```

Comment:
Don't use auto here since the type is not obvious based on the RHS only: https://llvm.org/docs/CodingStandards.html#use-auto-type-deduction-to-make-code-more-readable